MCprep_addon/materials/generate.py

```diff
@@ -436,7 +439,9 @@ def set_cycles_texture(
 				node.image = new_img
 				node.mute = False
 				node.hide = False
-				util.apply_colorspace(node, 'Non-Color')
+				res = util.apply_noncolor_data(node)
+				if res is not None:
+					env.log(f"TypeError on {res.line} in {res.file}: {res.err_type}")
```

Comment:
Since we use this in several places, maybe conf.py could provide a "static" const template string? Maybe or even a utility function called `log_error_lineno()` which takes this object as an input.